catalog/ui/catalog-ui-search/src/main/webapp/component/paging/paging.view.js

```diff
@@ -56,15 +58,24 @@ define([
         lastPage: function() {
             this.model.getLastPage();
         },
+        previousServerPage: function() {
+            this.options.query.getPreviousServerPage();
+        },
+        nextServerPage: function() {
+            this.options.query.getNextServerPage();
+        },        
         onRender: function(){
             this.updateSelectionInterface();
         },
         serializeData: function(){
+            var query = this.options.query;
```

Comment:
This can also be accessed through `this.options.selectionInterface.getCurrentQuery()`